pkg/cloud/libvirt/client/domain.go

```diff
@@ -36,7 +35,7 @@ type pendingMapping struct {
 	network  *libvirt.Network
 }
 
-func newDomainDef() libvirtxml.Domain {
+func newDomainDef(arch string) libvirtxml.Domain {
```

Comment:
This change's reason is that I think it may have an impact when targeting remote libvirtd instance, it related the xml which will be created for VM, and on the judgment of arch I think it should use getHostArchitecture not runtime. 
In fact, I don't know if "Graphics/Spice isn't supported on s390/s390x platform" is right, I see the code is first submitted in #174.
If you think it is not appropriate to put it here, I can put it in a new separate patch?

Reply:
> For the judgment of arch, I think it should use getHostArchitecture not runtime.

I didn't disagree with that. I'm talking about passing around arch specifically instead of each call using `getHostArchitecture`. In future, these functions might need access to other data from connection and if we pass connection, we won't need to add more parameters.

Reply:
>  If you think it is not appropriate to put it here, I can put it in a new separate patch?

Yes, ideally it should be a separate patch as its a separate logical change (will build and run fine without the config drive changes, can be depedency for other changes than this PR etc).

Reply:
ok, I see, I agree with you. so I will create a new PR to modify this, and I will delete this change about newDomainDef here.

Reply:
No separate PR needed for non-functional changes like this. Seperate patch/commit is warranted though.

Reply:
sorry, I misunderstood what you meant. I can independently submit another commit in this PR, right?

Reply:
I changed my code: 
pass "*libvirt.Connect" instead of arch returned by getHostArchitecture, and seperated the code as a new commit.
Is my understanding of your comments correct?

Reply:
> I can independently submit another commit in this PR, right?

Yes. It's just `git push -f` as usual. :)

>  Is my understanding of your comments correct?

Yes. Thanks so much.